ed25519_utils.go

```diff
@@ -14,7 +13,7 @@ var (
 )
 
 // ParseEdPrivateKeyFromPEM parses a PEM-encoded Edwards curve private key
-func ParseEdPrivateKeyFromPEM(key []byte) (crypto.PrivateKey, error) {
+func ParseEdPrivateKeyFromPEM(key []byte) (ed25519.PrivateKey, error) {
```

Comment:
Given these are public functions, and the underlying types are different, I think this would be a breaking change.

We avoid breaking changes in major versions.

I do agree having parity would be nice, but not at the expense of introducing an incompatible change.